languages/go/exercises/concept/maps/.docs/after.md

```diff
@@ -0,0 +1,58 @@
+Further Reading:
+
+- [godoc-race-detector]
+- [goblog-race-detector]
+- [goblog-iteration-order]
+
+A `map` in Go is an associative data type that contains a collection of key/value pairs, which other languages might refer to as as dictionary or an associative array.
```

Comment:
Is there maybe a document you can refer to that describes maps in Go?

Reply:
Fixed by 2bd7e2f